battleship-client/src/main/java/com/academy/solid/nie/client/ui/Cell.java

```diff
@@ -39,24 +39,22 @@ boolean wasShot() {
      * It changes the cell properties.
      *
      * @param newShip
-     * @return
      */
-    Cell addShip(final Ship newShip) {
+    void addShip(Ship newShip) {
```

Comment:
All methods which add an element to collection / object come in two flavours. You are switching flavours here. Why? Also, why no longer final?

Reply:
So we decided as a team that we are removing finals in parameters from our code.
Could you explain what do you mean by switching flavours? We don't need this.method to return Cell. We will handle it and rename to setShip, because basically it is just setting Ship to Cell.